mission/config/subconfigs/vehicle_respawn_info.hpp

```diff
@@ -913,7 +913,8 @@ class spawn_point_types {
 				name = "M113";
 				icon = VEHICLE_ICON_ARMOUR;
 				include[] = { { "m113", "transport" } };
-				exclude[] = { { "firesupport", "arvn", "pavn" } };
+				// Apparently this works, while {"firesupport", "arvn", "pavn"} does not
+				exclude[] = { { "firesupport" }, { "arvn" }, { "pavn" } };
```

Comment:
See comment

Reply:
You WOT mate?

Reply:
i dunno, but this worked and the one before didn't 🤷